bin/ghe-backup

```diff
@@ -65,7 +65,7 @@ if [ -f ../in-progress ]; then
   progress=$(cat ../in-progress)
   snapshot=$(echo "$progress" | cut -d ' ' -f 1)
   pid=$(echo "$progress" | cut -d ' ' -f 2)
-  if ! ps -p $pid -o command= | grep ghe-backup; then
+  if ! ps -p $pid -o comm= | grep ghe-backup; then
```

Comment:
What about `ps -p 103090 -o args`? according to https://docs.oracle.com/cd/E26502_01/html/E29030/ps-1.html, that may work on Solaris too?